### ISSUE TITLE
Partition-less health monitor

### ISSUE THREAD

**Issue by vercellone** [joel74/POSH-LTM-Rest#131]
@elijahgagne reported that health monitors created in his environment do not include a partition property.  Refer to his comments May 3-4, 2016 on #55 for details.

**Comment by elijahgagne** [comment]
May 3-4, 2016 :)

**Comment by vercellone** [comment]
Corrected.

**Comment by joel74** [comment]
Doing some testing on this. This works fine for me - requesting health monitors by partition returns only those, and 'Partition' is a property returned.

On 11.6, I successfully retrieve health monitors when passing in:
1) partition and name
2) partition and application
3) partition, name and application
(NB: Partition value was always 'test' - 'common' wasn't used.)
I also successfully created a new health monitor in the Test partition. 

Elijah, could you shed some light on whether you're still experiencing the issue and if so, how it can be repro'd?

Thanks, Joel

**Comment by elijahgagne** [comment]
I am unable to reproduce this anymore. The server response always contains a short name and a partition. 

I've tried creating several iApps from different templates (including the template I know I used when I originally reproduced this issue), but the result was the same in all tests. I'm not sure what changed, but I do know we have made multiple minor upgrades to the LTMs in the last year+. 

Sorry for the extra work. I'm good with closing this issue out. Thanks.
